Fuzzlyn/Execution/ProgramExecutor.cs

```diff
@@ -85,7 +85,7 @@ public static List<ProgramPairResults> RunSeparately(List<ProgramPair> programs)
             ProcessStartInfo info = new ProcessStartInfo
             {
                 FileName = dotnet,
-                Arguments = $"\"{fuzzlyn}\" --execute-programs",
+                Arguments = " --execute-programs",
```

Comment:
We should try to keep this working for the host case. Here is what I suggest:
1. Rename `dotnet` to `host`
2. Compare `host` with `fuzzlyn`: if they are equal, FileName = fuzzlyn, Arguments = --execute-programs. If they are not equal, FileName = host, Arguments = what it used to be.

This should let it keep working when running this on custom builds on CoreCLR.

Reply:
Good point, I didn't consider that scenario. I've made these changes, let me know what you think.